webpack.config.js

```diff
@@ -27,21 +27,6 @@ module.exports = () => {
     },
     module: {
       rules: [
-        {
-          test: /lib\/.+\.jsx?$/,
-          exclude: /node_modules|lib\/simperium/,
-          enforce: 'pre',
-          use: [
-            {
-              loader: 'eslint-loader',
```

Comment:
This is likely a dependency that can be removed as well.

Reply:
Thanks. I'll update.

Reply:
removed dependency in 065321e